.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+  - package-ecosystem: "cargo"
+    directory: "/"
```

Comment:
We need this to also run for the crates under `internal/`

Reply:
To be clear, this is the only repo that has this problem.

Reply:
I've just added a second `cargo` entry with a second directory path. I think that's how you're supposed to do it.